releases.md

```diff
@@ -29,7 +31,31 @@ For more information on the changes planned for each release, please see the [Pr
 | 1.3.6          | September 30, 2022 | October 6, 2022    |
 | 2.4            | November 3, 2022   | November 10, 2022  |
 | 1.3.7          | December 1, 2022   | December 8, 2022   |
-| 2.5            | January 10, 2023   | January 12, 2023   |
+| 2.5            | January 10, 2023   | January 17, 2023   |
```

Comment:
You can just remove this since its moved to the 2023 section.

Reply:
Removed